source/Halibut/Transport/SecureListeningClient.cs

```diff
@@ -128,7 +128,7 @@ public async Task ExecuteTransactionAsync(ExchangeActionAsync protocolHandler, R
                     // against all connections in the pool being bad
                     if (i == 1)
                     {
-                        await connectionManager.ClearPooledConnectionsAsync(ServiceEndpoint, log, requestCancellationTokens.InProgressRequestCancellationToken);
+                        await connectionManager.ClearPooledConnectionsAsync(ServiceEndpoint, log, requestCancellationTokens.LinkedCancellationToken);
```

Comment:
What was the motivation for this change?

Reply:
Good question, I think it was a hangover from combining the tokens in my head and this just looked wrong. I'll revert though as it's not the intent to do this in this PR!!

Reply:
Removed